### PR TITLE
Add `isLive`, fix `creators` and refactor code

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
See commit messages for more information. Most stuff is just simple code refactor stuff. Regarding functionality, it just adds `isLive` to events and fixes `creator` to `creators`. Review commit by commit.

### CODE REVIEW

**Review by JulianKniephoff — COMMENTED** [review]
Just some nitpicks

File: modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java [hunk 1]
```diff
@@ -97,8 +97,8 @@ class Item {
           .collect(Collectors.toCollection(ArrayList::new));
 
       // Assemble ACL
-      final List<Jsons.Val> canReadRoles = new ArrayList();
-      final List<Jsons.Val> canWriteRoles = new ArrayList();
+      final List<Jsons.Val> canReadRoles = new ArrayList<Jsons.Val>();
+      final List<Jsons.Val> canWriteRoles = new ArrayList<Jsons.Val>();
```
[the remark this answers is not in the document]
Comment (LukasKalbertodt) [reply modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java]: This is outdated, I am using `var` now.

File: modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java [hunk 2]
```diff
@@ -111,14 +111,18 @@ class Item {
           Jsons.p("write", Jsons.arr(canWriteRoles))
       );
 
+      final List<Jsons.Val> creators = Arrays.stream(mp.getCreators())
```
[the remark this answers is not in the document]
Comment (LukasKalbertodt) [reply modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java:119]: Mh, I am pretty sure I tried it but it didn't work. But oh well, this seems to work.

File: modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (JulianKniephoff) [reply modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java]: right :ok_hand:

File: modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (JulianKniephoff) [reply modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java]: Oh interesting. :thinking: 

But if it works now ... :shrug:

**Pull request merged**